.github/workflows/sync.yml

```
@@ -0,0 +1,18 @@
+name: Sync Files
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  convert-test-sync:
+    name: Sync PCL test files
+    runs-on: macos-latest
```

We have more Ubuntu runner capacity than macOS

```suggestion
    runs-on: ubuntu-latest
```